### PR TITLE
Correct homepage code examples

### PR DESCRIPTION
**Pull request by nickhudkins**
The examples for Extensibility on the home page are broken :(.

RCT_EXPORT_MODULE() is missing, and RCT_EXPORT_VIEW_PROPERTY's signature has changed to accept a type as the 2nd argument.

### CODE REVIEW

**Comment by brentvatne** [comment]
@nickhudkins - can you also add this change to the `README`?

**Comment by nickhudkins** [comment]
@brentvatne done!

File: README.md [hunk 1]
```diff
@@ -177,6 +177,8 @@ Custom iOS views can be exposed by subclassing `RCTViewManager`, implementing a
 
 @implementation MyCustomViewManager
 
+RCT_EXPORT_MODULE();
```
Comment (brentvatne) [inline README.md]: Incredibly small nit - the convention throughout the product leaves out the semicolon after using these macros, could we remove that?

**Comment by brentvatne** [comment]
Thanks @nickhudkins! Sorry for the delay

**Pull request merged**